pkg/security/resolvers/tags/resolver.go

```diff
@@ -53,17 +54,17 @@ func (t *DefaultResolver) Resolve(id string) []string {
 
 // ResolveWithErr returns the tags for the given id
 func (t *DefaultResolver) ResolveWithErr(id string) ([]string, error) {
-	return GetTagsOfContainer(t.tagger, id)
+	return GetTagsOfContainer(t.tagger, containerutils.ContainerID(id))
```

Comment:
Why not changing the type of id in ResolveWithErr instead ?

Reply:
You're right. Changing it